ckanext/collaborators/tests/test_functional.py

```diff
@@ -0,0 +1,302 @@
+from nose.tools import assert_in, assert_not_in, assert_equal
+
+from ckan.tests import helpers, factories
+from ckan.plugins import toolkit
+
+submit_and_follow = helpers.submit_and_follow
+
+class TestCollaborators(helpers.FunctionalTestBase):
+
+    def setup(self):
+
+        super(TestCollaborators, self).setup()
+
+        helpers.reset_db()
```

Comment:
The parent `setup()` method takes care of this so this line should not be necessary